src/components/templates/Camp/components/Map.tsx

```diff
@@ -18,22 +19,31 @@ export const Map: React.FC<PropsType> = ({
   setSelectedRoom,
   setIsRoomModalOpen,
 }) => {
-  const openRoomModal = useCallback(
-    (room: CampRoomData) => {
-      setSelectedRoom(room);
-      setIsRoomModalOpen(true);
-    },
-    [setSelectedRoom, setIsRoomModalOpen]
-  );
+  // const openRoomModal = useCallback(
+  //   (room: CampRoomData) => {
+  //     setSelectedRoom(room);
+  //     setIsRoomModalOpen(true);
+  //   },
+  //   [setSelectedRoom, setIsRoomModalOpen]
+  // );
+  const [roomClicked, setRoomClicked] = useState("");
+  const [clicked, setClicked] = useState(false);
 
   if (!venue) {
     return <>Loading map...</>;
   }
 
+  const rooms: CampRoomData[] = JSON.parse(JSON.stringify(venue.rooms));
+  if (roomClicked !== "") {
+    const idx = rooms.findIndex((room) => room.title === roomClicked);
```

Comment:
good to check for idx === -1

Reply:
what do you mean?

Reply:
look at the documentation for findIndex. What happens if it can't find what you're looking for?